circle.yml

```diff
@@ -6,15 +6,23 @@ general:
       - production
 machine:
   post:
-    - npm install jscs@1.13.1 -g
-    - npm install mocha@2.4.5 -g
+    - npm install jscs@1.13.1 mocha@2.4.5 -g
+    - cd api && npm run test:e2e website install
```

Comment:
`frontend` in my view is synonymous for "client", e.g website and app are the front-ends, API is the backend. So `test:frontend` would mean running all client tests, including unit tests, no? In my view nightwatch tests are not "front-end" tests because they cover both the front-end and back-end, i.e. they are end-to-end. But I'm the new one in web development so please correct me...

Nightwatch's doc seems to say the same: "Nightwatch.js is an easy to use Node.js based End-to-End (E2E) testing solution for browser based apps and websites." http://nightwatchjs.org/

I can change to `nightwatch` otherwise? I just prefer to avoid technology names where possible.